### PR TITLE
DOCUMENTATION: Proccessings Proofing

### PR DESCRIPTION
**Pull request by glhays**
Closes: #286

### CODE REVIEW

File: 2. Services/2.2 Processings/2.2 Processings.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Here's an example of what a Processing service function would look like:
 public ValueTask<Student> UpsertStudentAsync(Student student) =>
 TryCatch(async () =>
 {
-    ValidateStudent(student);
+    ValidateStudentOnUpert(student);
```
Comment (hassanhabib) [inline 2. Services/2.2 Processings/2.2 Processings.md:15]: `OnUpsert`

File: 2. Services/2.2 Processings/2.2 Processings.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ TryCatch(async () =>
         retrievedStudent.Id == student.Id);
 
     return studentExists switch {
-        false => await this.studentService.RegisterStudentAsync(student),
+        false => await this.studentService.AddStudentAsync(student),
```
Comment (hassanhabib) [inline 2. Services/2.2 Processings/2.2 Processings.md:24]: This should stay as `RegisterStudentAsync` the idea here is that the language is more business specific than technical at the `Processing` layer.

File: 2. Services/2.2 Processings/2.2 Processings.md (hunk 1 above)
Comment (glhays) [reply 2. Services/2.2 Processings/2.2 Processings.md:15]: Good eyes!

Comment (glhays) [reply 2. Services/2.2 Processings/2.2 Processings.md]: Fixed

File: 2. Services/2.2 Processings/2.2 Processings.md (hunk 2 above)
Comment (glhays) [reply 2. Services/2.2 Processings/2.2 Processings.md]: Fixed

**Pull request merged**